packages/plugin-ext/src/hosted/browser/hosted-plugin.ts

```diff
@@ -217,6 +217,7 @@ export class HostedPluginSupport {
                 };
             }
         });
+        this.activateByEvent('onStartupFinished');
```

Comment:
I was wondering why the event should be sent at this location and not another. What parts of the system need to be ready in order for startup to be finished? Not saying it's the wrong place, I just want to understand your thinking.